lib/build-view.js

```diff
@@ -190,9 +186,23 @@ export default class BuildView extends View {
     };
     const orientation = atom.config.get('build.panelOrientation') || 'Bottom';
     this.panel = addfn[orientation].call(atom.workspace, { item: this });
+    if (atom.config.get('build.hidePanelHeading') === false) {
+      this.resizeHeading();
+    }
+    this.fixTerminalElHeight();
     this.resizeToNearestRow();
   }
 
+  resizeHeading() {
+    const headingHeight = $('.item-views').position().top;
+    $('.build .heading').css('height', `${headingHeight}px`);
+  }
```

Comment:
Its not strictly necessary, I just think it just looks neater with the themes I have tested with as it makes the build output panel line up with the editor view when the panel is left or right.
![atom-material-ui](https://cloud.githubusercontent.com/assets/5864491/14930585/fb297fc2-0e5c-11e6-814e-6157f18901a8.png)
![atom-dark-ui](https://cloud.githubusercontent.com/assets/5864491/14930588/02184340-0e5d-11e6-8938-db2b8763e8c2.png)